share/availability/window.go

```diff
@@ -11,8 +14,33 @@ const (
 // given AvailabilityWindow. If the window is disabled (0), it returns true for
 // every timestamp.
 func IsWithinWindow(t time.Time, window time.Duration) bool {
+	if windowOverride {
+		window = windowOverrideDur
+	}
 	if window == time.Duration(0) {
 		return true
 	}
 	return time.Since(t) <= window
 }
+
+// windowOverride is a flag that overrides the availability window. This flag is intended for
+// testing purposes only.
+var (
+	windowOverride    bool
```

Comment:
Nit: not needed, `windowOverrideDur != 0` will be enough.

Reply:
Zero window override is a valid case and the one i need. It disables the whole pruning

Reply:
a-ha

Reply:
So if override is 0 and the window is 10, the window will stay 10, even if i need as 0